test/fixtures/dialects/bigquery/load_data_statement.sql

```diff
@@ -0,0 +1,106 @@
+LOAD DATA INTO mydataset.table1
```

Comment:
*note:* I copied all examples from [BQ docs website](https://cloud.google.com/bigquery/docs/reference/standard-sql/load-statements#examples).

Reply:
*question:* do we care here to preserve the dialect logic? The example below will parse but fail in BigQuery.

```sql
-- This query returns an error.
LOAD DATA INTO mydataset.table1
  (
    x INT64, -- column_list is given but the partition column list is missing
    y STRING
  )
  FROM FILES(
    format='AVRO',
    uris = ['gs://bucket/path/*'],
    hive_partition_uri_prefix='gs://bucket/path'
  )
  WITH PARTITION COLUMNS
```

I assume as a parser we do not care about dialect logic. But want to make sure.